package.json

```diff
@@ -11,7 +11,9 @@
     "dev": "webpack -w",
     "precommit": "lint-staged",
     "postinstall": "node download/install.js",
-    "pack": "webpack && pkg dist/now.js -c package.json -o packed/now --options no-warnings",
+    "gzip": "ls packed/now* | xargs gzip -k",
+    "bundle": "pkg dist/now.js -c package.json -o packed/now --options no-warnings",
+    "pack": "rm -rf packed && webpack && npm run bundle && npm run gzip",
```

Comment:
This makes it harder to test locally – you'd need to `ungzip`

Reply:
do you?

Reply:
No worries, @matheuss. The `-k` flag for `gzip` ensures the unzipped binaries still stay around locally, so they don't get replaced.

Besides of that, you only need to double click `.gz`s on macOS in order to unzip them.